set.mm

```diff
@@ -153948,6 +154055,29 @@ are used instead of sets because their representation is shorter (and more
       DXRYEUVEUVFXPXSXQBCDAUVJUVGYEUVK $.
   $}
 
+  $( The number of elements of a finite function restricted to a subset of its
+     domain.  (Contributed by AV, 15-Dec-2021.) $)
```

Comment:
The number of elements of a finite function restricted to a subset of its domain is equal to the number of elements of that subset.

Reply:
OK, I will change that.